model/src/olivia.rs

```diff
@@ -10,10 +10,18 @@ use time::macros::format_description;
 use time::OffsetDateTime;
 use time::PrimitiveDateTime;
 use time::Time;
+use url::Url;
 
 pub const EVENT_TIME_FORMAT: &[FormatItem] =
     format_description!("[year]-[month]-[day]T[hour]:[minute]:[second]");
 
+#[derive(Debug, Clone, Deserialize, PartialEq)]
+pub struct Attestation {
+    pub id: BitMexPriceEventId,
+    pub price: u64,
+    pub scalars: Vec<SecretKey>,
+}
+
```

Comment:
Separate commit?

Reply:
Perhaps yeah. Ties in with what I commented above. Up to you!